src/test/run-make/unicode-input/Makefile

```diff
@@ -1,26 +0,0 @@
--include ../tools.mk
-
-# This test attempts to run rustc itself from the compiled binary; but
-# that means that you need to set the LD_LIBRARY_PATH for rustc itself
-# while running multiple_files, and that won't work for stage1.
-
-# FIXME ignore windows
-ifndef IS_WINDOWS
-ifeq ($(RUST_BUILD_STAGE),1)
-DOTEST=
-else
-DOTEST=dotest
-endif
-endif
-
-all: $(DOTEST)
-
-dotest:
-	# check that we don't ICE on unicode input, issue #11178
```

Comment:
It's redundant with the UI unicode test (which tests the same thing)

That said, there's something odd with how unicode is handled by both new and old mode, so that's worth exploring (though that exploration shouldn't block this but rather be something on our todo list)